### PR TITLE
create_basic_vm sample added

### PR DESCRIPTION
**Pull request by kumahesh**

### CODE REVIEW

File: samples/create_basic_vm.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+
+def get_args():
+    parser = cli.build_arg_parser()
+
+    parser.add_argument('-vn', '--vm-name', required=True,
+                        help="Name of the VirtualMachine to create.")
+    parser.add_argument('-hn', '--host-name', required=True,
+                        help="Name of the host where vm will reside")
+    parser.add_argument('-dn', '--datastore-name', required=True,
```
Comment (pgbidkar) [inline samples/create_basic_vm.py]: is it possible to make datastore name optional. If user doesnt pass datastore name, can we retrieve it from hostname provided?

File: samples/tools/pchelper.py [hunk 2]
```diff
@@ -99,3 +99,13 @@ def get_container_view(service_instance, obj_type, container=None):
         recursive=True
     )
     return view_ref
+
+
+def get_obj(si, vimtype, name):
```
Comment (pgbidkar) [inline samples/tools/pchelper.py]: I think we have too many get_obj() methods. Move this to sample.

File: samples/create_basic_vm.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
+# Description: This sample creates basic virtual machine
```
Comment (pgbidkar) [inline samples/create_basic_vm.py]: Please can you add Author and Copyright info here?

Comment (kumahesh) [reply samples/create_basic_vm.py]: added. please check

**Comment by kumahesh** [comment]
Duplicate. Already we have a sample for creating vms https://github.com/vmware/pyvmomi-community-samples/blob/master/samples/create_random_marvel_vms.py

**Pull request closed without merging**